### ISSUE TITLE
Add support for accessing system / host information in config

### ISSUE THREAD

**Issue by ekamil** [johnste/finicky#138]
Hi,
thanks for finicky :)! I'm trying to use a single .finicky.js accross multiple macs. Given that Chrome doesn't name profile dirs sensibly I wanted to have a map hostname => Profile directory.
But I keep getting `Configuration: ReferenceError: Can't find variable: os`
I can trigger this error with "https://www.youtube.com/watch" in finicky console and with the config below.

Example minimal config:
```
profiles = {
  "Kamils-MacBook-Pro.local": {
    chrome: {
      main: {
        name: "Google Chrome",
        profile: "Profile 2",
        openInBackground: false,
      },
    },
  },
};

module.exports = {
  defaultBrowser: "Safari",
  handlers: [
    {
      match: ["*google.com*", "*youtube.com*", "*youtu.be*"],
      browser: () => {
        profiles[os.hostname()].chrome.main;
      },
    },
    {
      match: /zoom.us\/j\//,
      browser: "us.zoom.xos",
    },
  ],
  rewrite: [],
};
````

**Comment by johnste** [comment]
Added `finicky.getSystemInfo` in b55646d3a9231846a47886f1690bc78898fd39c6, will try to have a new release out this weekend.

**Comment by johnste** [comment]
Removed address since it was reporting random ip addresses and didn't seem that useful.

This feature is available in [3.3.0](https://github.com/johnste/finicky/releases/tag/v3.3.0)
